### ISSUE TITLE
Autocomplete for '\begin{}' inserts a double closing '}'

### ISSUE THREAD

**Issue by albanleandri** [James-Yu/LaTeX-Workshop#2075]
### Disable all the other extensions except for LaTeX Workshop, and check that you still see this issue. [Required]

You still see this issue?: Yes

### Make sure to visit the [wiki FAQ](https://github.com/James-Yu/LaTeX-Workshop/wiki/FAQ) before filling an issue.

You visited the wiki?: Yes

### If your issue is with compiling a document (not having to do with finding the root file of a project), check first that you can compile manually.

You can compile a TeX document manually?: Yes

## Describe the bug [Required]
Invalid syntax from autocomplete for \begin{itemize} or \begin{enumerate} causing compilation error.

### To Reproduce
When typing:
```\begin{i}```
The suggestions for autocomplete will appear. 
In this case, the relevant suggestion will be:
```\begin{itemize}```
If I proceed with this suggestion then I will have what follows:

```
\begin{itemize}
    \item 
\end{itemize}}
```

As you can see, the ending command contains an additional closing curly brace.
It's wrong and will cause a compilation error if not removed (manually).

### Expected behavior

I want the autocomplite to lead to what's below:
```
\begin{itemize}
    \item 
\end{itemize}
```

## Logs [Required]

Please paste the whole log messages here, not parts of ones. It is very important to identify problems.

### LaTeX Workshop Output [Required]

```
To access the log, click the 'TeX' icon on the Activity Bar on the left side, select 'View Log Messages', then select 'View LaTeX Workshop extension log'.

[Paste the log here]
```

### Developer Tools Console [Required]

```
To access the log, click 'help' -> 'Toggle Developer Tools. Paste anything suspicious.

[Paste the log here]
```

## Screenshots

![image](https://user-images.githubusercontent.com/23435414/81475312-9f1d1400-920b-11ea-8485-7b21d644600b.png)

![image](https://user-images.githubusercontent.com/23435414/81475343-c7a50e00-920b-11ea-8c41-bcc4285828b1.png)

## Desktop [Required]
 - OS: Windows 10
 - VS Code version: [e.g. 1.45.0]
 - Extension version: [e.g. 8.9.0]
 - TeX distribution version: [e.g. TeX Live 2020]

### Are you using LaTeX Workshop with VS Code Remote?

Yes, I am using LaTeX Workshop

**Comment by jlelong** [comment]
You should not write the closing `}` of `\begin` your self.

![May-09-2020 15-52-06](https://user-images.githubusercontent.com/2910140/81475517-130bec00-920d-11ea-8abc-1d2f53373b47.gif)

**Comment by albanleandri** [comment]
@jlelong actually, I don't, it's auto closing.
Is it a VSC setting I can change?

**Comment by jlelong** [comment]
OK I understand. I only type `\beg` and hit `TAB` to complete. Doing I only get `\begin{`. I agree with you that when typing `\begin{` you automatically get the closing `}`.

I will investigate if I can remove the extra `}` on the fly if it has already been inserted.

**Comment by jlelong** [comment]
I think this is solved by #2080 
Could you test this branch to make sure it does not have side effects?
